commands/Discord/help.js

```diff
@@ -0,0 +1,63 @@
+const Discord = require('discord.js');
+
+module.exports = {
+  name: 'help',
+  description:
+      'Shows help for a specific command, or lists all available commands.',
+  usage: '[command]',
+  aliases: ['commands'],
+  category: 'Discord',
+
+  execute(message, args) {
+    const embed = new Discord.MessageEmbed()
+      .setTitle('FossBot Help')
+      .setColor('#04c46c')
+      .attachFiles(['logo.png'])
+      .setThumbnail('attachment://logo.png')
+      .setAuthor(
+        'FossBot🤖',
+        'attachment://logo.png',
+        'https://github.com/fossgect/fossbot/',
+      );
+
+    const reply = [];
+    const { commands } = message.client;
+
+    if (!args.length) {
```

Comment:
check for more than 1 args
```suggestion
    if (!args.length || args.length > 1) {
```

Reply:
But then if someone uses `!help invite gh`, instead of showing the help for the first arg (if it's valid), the list of commands would be displayed. :-(